client/src/components/Header_Bar/header.css

```diff
@@ -1,18 +1,19 @@
 .HeadBar {
   position: absolute;
-  width: 375px;
+  width: 100%;
   height: 57px;
   left: 0px;
   top: 0px;
 
```

Comment:
Please lint :fire:

Reply:
we tested our product on Samsung 6 edge and iphone 8 ,
and it's working , i don't say perfectly . we need to refactor later it's MVP for now.
and only for MOBILE .
we will try to use only percents in the future as you mentioned , to make it fit also for tablets.